LinkedList/Exercise/SLList.h

```diff
@@ -0,0 +1,328 @@
+#pragma once
+
+template<typename Type>
+class TSLList
+{
+private:
+
+	struct FNode
+	{
+		Type Element;
```

Comment:
THIS is just my humble personal opinion... to call what's inside the FNode "Element" isn't so clear... The element of a node isn't really clear. Something like Data might say a lot more.

Reply:
I agree